prow/plank/README.md

```diff
@@ -0,0 +1,41 @@
+# Plank
+
+Plank is the controller that manages the job execution and lifecycle for jobs running in k8s.
+
+### Usage
+```bash
+$ bazel run //prow/cmd/plank -- --help
+```
+
+### Configuration
+The following job template is based on the format for `spyglass`.
```

Comment:
What does this mean?

Reply:
Looking at Openshift's and test-infra's `config.yaml`, the template URL is based on `gubernator` and there were a lot of `if else` statements, but `spyglass` seems to be pretty simple for just thought of clarifying that in case someone got confused when referencing other existing prow's `config.yaml`

Reply:
Oh -- ok. This is only a useful comment for someone who is running Gubernator. As this doc will help new deployments get set up, let's forget the comment and focus on making it clear to use the spyglass link in that template

Reply:
okay, got it. i'll delete it then.